builders/manage/sudo/minting.md

```diff
@@ -0,0 +1,64 @@
+---
+
+# Using Sudo to Mint Native Tokens
+
+## Introduction {: #introduction }
+
+[Sudo](https://paritytech.github.io/polkadot-sdk/master/pallet_sudo/index.html){target=\_blank} is a module that enables privileged runtime calls to be dispatched when called from the Sudo account. Sudo is sometimes colloquially referred to as a superuser or god-like account. This enables you to perform privileged actions in the course of managing your appchain, such as minting new native tokens. 
+
+In this guide, you'll learn how to use Sudo to properly mint new native tokens. This comprehensive guide shows how to check the balance of an existing account prior to assigning it a new balance with Sudo access. 
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+ - A Tanssi Appchain (Snap or Dedicated)
+ - Your Appchain's Sudo account connected to your Appchain's Polkadot.js Apps. You can refer to the [Managing Sudo guide](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank} for instructions on injecting your Sudo account into Polkadot.js Apps
+
+If you're unsure what your Tanssi Appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/minting/minting-1.webp)
+
+!!! note
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi Appchain.
+
+## Minting Tokens {: #minting-tokens }
+
+As you know, the Sudo account has the ability to perform privileged functions, including minting additional tokens. When setting up your Tanssi Appchain on [apps.tanssi.network](https://apps.tanssi.network/){target=\_blank}, you can specify genesis account balances. In other words, you have the ability to endow accounts with initial balances upon launching your Tanssi Appchain. However, you can also mint new tokens after launch with the help of the Sudo account.
+
+!!! note
+    This tutorial demonstrates assigning arbitrary token balances on a TestNet Appchain that has no value. You should carefully consider the ramifications of creating additional tokens on your own Tanssi Appchain.
+
+### Checking Existing Account Balance {: #checking-existing-account-balance }
+
+The next section will demonstrate how to assign arbitrary token balances to accounts using the Sudo account. This process will overwrite the specified account's existing balance, so verifying the account is empty is a good practice before continuing. To check an account's balance, take the following steps:
+
+1. Navigate to the **Developer** tab of [Polkadot.js Apps](#configuring-polkadotjs-apps) and click on **Chain State**
```

Comment:
There's a couple of these links, but they need to be updated to point to the page where this section is now